eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -110,7 +110,7 @@ jobs:
       - pwsh: >
           dotnet test eng/service.proj
           --framework $(TestTargetFramework)
-          --filter "TestCategory!=Manually"
+          --filter "(TestCategory!=Manually)&(FullyQualifiedName~Azure.Identity.Tests.VisualStudioCodeCredentialLiveTests)"
```

Comment:
Is adding this to the shared test template the only way to exclude this? Why can't we just disable the test in source?